commands/deploy.go

```diff
@@ -26,8 +26,8 @@ const MaxFileSize = 1073741824 // 1GB
 
 // DeployCmd handles deploying an app to Section.
 type DeployCmd struct {
-	AccountID        int `default:"4322"`  // harc-coded until authentication is implemented
-	AppID            int `default:"65443"` // hard-coded for now until authentication is implmented
+	AccountID        int `required`
```

Comment:
Tools like the GCloud SDK allow you to specify default values in the user's configuration file to avoid passing the project-id every time. Similar to AWS CLI storing the default region. Should we consider this?

Reply:
Yep, and I even have a half written PR for setting defaults. 

Need to get that published and released in this week!